lib/ansible/plugins/lookup/aws_service_ip_ranges.py

```diff
@@ -0,0 +1,75 @@
+# (c) 2016 James Turner <turnerjsm@gmail.com>
+# (c) 2017 Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = """
+lookup: aws_service_ip_ranges
+author:
+  - James Turner <turnerjsm@gmail.com>
+version_added: "2.5"
+requirements:
+  - must have public internet connectivity
```

Comment:
i spent too much time debating this in my head, it should be obvious .. .but my soul darkens a bit realizing I WILL inevitably have to point to this doc when someone w/o connectivity complains about it not working ...

Reply:
Hehe... well at least open_url will handle proxies consistently. I can delete it, but I figured it doesn't hurt anything to have it in there.